docs/reference/esql/esql-kibana.asciidoc

```diff
@@ -2,14 +2,240 @@
+button (image:images/esql/esql-icon-expand-query-bar.svg[]) to expand the query
+bar:
+
+image::images/esql/esql-expanded-query-bar.png[align="center"]
+
+To return to a compact query bar, click the minimize editor button 
+(image:images/esql/esql-icon-minimize-query-bar.svg[]).
+
+[discrete]
+==== Warnings
+
+A query may result in warnings, for example when querying an unsupported field
+type. When that happens, a warning symbol is shown in the query bar. To see the
+detailed warning, expand the query bar, and click *warnings*.
+
+[discrete]
+[[esql-kibana-results-table]]
+=== The results table
+
+For the example query, the results table shows 10 rows. Omitting the `LIMIT`
+command, the results table defaults to up to 500 rows. Using `LIMIT`, you can
+increase the limit to up to 10,000 rows.
+
+Each row shows two columns for the example query: a column with the `@timestamp`
+field and a column with the full document. To display specific fields from the
+documents, use the <<esql-keep>> command:
+
+[source,esql]
+----
+FROM kibana_sample_data_logs
+| KEEP @timestamp, bytes, geo.dest
+----
+
+To display all fields as separate columns, use `KEEP *`:
+
+[source,esql]
+----
+FROM kibana_sample_data_logs
+| KEEP *
+----
+
+NOTE: The maximum number of columns in Discover is 50. If a query returns more
+than 50 columns, Discover only shows the first 50.
+
+[discrete]
+==== Sorting
+
+To sort on one of the columns, click the column name you want to sort on and
+select the sort order. Note that this performs client-side sorting. It only
+sorts the rows that were retrieved by the query, which may not be the full
+dataset because of the (implicit) limit. To sort the full data set, use the
+<<esql-sort>> command:
+
+[source,esql]
+----
+FROM kibana_sample_data_logs
+| KEEP @timestamp, bytes, geo.dest
+| SORT bytes DESC
+----
+
+[discrete]
+[[esql-kibana-time-filter]]
+=== Time filtering
+
+To display data within a specified time range, use the
+{kibana-ref}/set-time-filter.html[time filter]. The time filter is only enabled
+when the indices you're querying contain a field called `@timestamp`. If your
+indices do not contain an `@timestamp` field, limit the time range using the
+<<esql-where>> command instead.
+
+[discrete]
+[[esql-kibana-visualizations]]
+=== Analyze and visualize data
+
+Between the query bar and the results table, Discover shows a date histogram
+visualization. If the indices you're querying do not contain an `@timestamp`
+field, the histogram is not shown.
+
+The visualization adapts to the query. A query's nature determines the type of
+visualization. Changing the previous example query to aggregate the total number
+of bytes per destination country:
+
+[source,esql]
+----
+FROM kibana_sample_data_logs
+| STATS total_bytes = SUM(bytes) BY geo.dest
+| SORT total_bytes DESC
+| LIMIT 3
+----
+
+changes the visualization into a bar chart showing the top 3 countries:
```

Comment:
I think you're correct not to capitalize this 'c' in 'change' here, but maybe we should get a second opinion.

Reply:
I wasn't sure about this one. It didn't look right, so I have changed it. Thanks!